b.c

```diff
@@ -863,22 +864,24 @@ bool fnematch(fa *pfa, FILE *f, char **pbuf, int *pbufsize, int quantum)
 	 * j: current character
```

Comment:
The first lines of this comment block should probably be reverted to their state before 904af5677d1886c947d0cd78fa63f739a8c9a394 too.  E.g.
```
  * All indices relative to buf.
  * i <= j <= k <= bufsize
```